src/ParametricOptInterface.jl

```diff
@@ -835,25 +843,113 @@ function MOI.optimize!(model::ParametricOptimizer)
         end
         empty!(model.updated_parameters)
     end
+
     MOI.optimize!(model.optimizer)
+
+    if model.evaluate_duals
+        calculate_dual_of_parameters(model)
+    end
 end
 
 ### Duals
 
-function MOI.get(model::ParametricOptimizer, attr::MOI.ConstraintDual, ci::MOI.ConstraintIndex)
-    return MOI.get(model.optimizer, attr, ci)
+function create_param_dual_cum_sum(keys_model_parameters::Base.KeySet{MOI.VariableIndex, Dict{MOI.VariableIndex, T}}) where T
+    param_dual_cum_sum = Dict{Int, Float64}()
+
+    for vi in keys_model_parameters
+        param_dual_cum_sum[vi.value] = 0.0
+    end
+    
+    return param_dual_cum_sum
+end
```

Comment:
In my opinion, you can change this function a little bit.

I think that 
```
create_param_dual_cum_sum(model::ParametricOptimizer)
    param_dual_cum_sum = Dict{Int, Float64}()
    for vi in keys(model.parameters)
        param_dual_cum_sum[vi.value] = 0.0
    end
    return param_dual_cum_sum
end
```
makes the function easier to understand.